.github/README.md

```diff
@@ -58,7 +58,16 @@ will keep committing.
 ```sh
 python contribute.py --days_before=10 --days_after=15
 ```
+Use `--start_date` to specify start date of commits. This option doesn't work if used with days_before.
```

Comment:
It shouldn't.
Imagine you need to set a period of time in last year.

Reply:
I mean, if someone uses both `--days_before` and `--start_date` this should lead to an error

Reply:
I think it is better when the logic is explicit and the user doesn't have to guess what would happen when they specify two parameters that collide

Reply:
I vote `--days_before` be removed altogether or just error out on if both are specified and if `--days_before` is removed also want to remove `--days_after` of course. I think specifying a date is the better route for the tool since it's more intrinsically what a user expects for options. Additionally it allows you to splice up your commit dates uniquely so the graph can be a bit more chaotic where `--days_before` and especially `--days_after` I find fairly useless.

And of course it would break backwards compatibility if anyone cares about that 😄